src/site/filters/liquid.js

```diff
@@ -709,6 +709,16 @@ module.exports = function registerFilters() {
     return string.replace(regex, newVal);
   };
 
+  liquid.filters.filterByTypeOfCare = (data, typeOfCare) => {
+    return data
+      .filter(
+        e =>
+          e.entity?.fieldServiceNameAndDescripti?.entity
+            ?.fieldVetCenterTypeOfCare === typeOfCare,
```

Comment:
Very cool that you got this working! However, this solution is specific for vet centers, since it uses `fieldVetCenterTypeOfCare`. Maybe we can provide the path to the typeOfCare attribute as an argument; then the function would be more generically useful.